src/inbox/Report.js

```diff
@@ -107,7 +112,13 @@ const Report = ({ reportId }) => {
               <span>Reported </span>
               {formatDistance(report.created_at, new Date())}
               <span> to </span>
-              <a href="#">{team.name}</a>
+              <Button
+                size="sm"
+                color="link"
+                onClick={event => dispatch({ type: actions.CHANGE_TEAM, payload: team.id })}
```

Comment:
I'm surprised eslint doesn't complain about unused variables

Reply:
I think it does complain :)